IFC4x3/Properties/n/NominalThickness_3cTq6sD71BVAf9l72L6lIy/Documentation.md

```diff
@@ -0,0 +1 @@
+The nominal thickness of the pavement. The size information is provided in addition to the shape representation and the geometric parameters used within. In case of inconsistency between the geometric parameters and the size properties, provided in the attached property set, the geometric parameters take precedence. It is regarded as constant thickness of the pavement, unless property ThicknessEnd is provided, in which case Thickness shall be regarded as the value at the beginning of a transition (usually at the object placement location). Example of such case is a (road) transition segment where the pavement object’s linear placement along an alignment denotes the beginning location.
```

Comment:
@larswik, is this a new property? is it worth generalising the NominalThickness property (so it can be used in other cases) and adding the Clarifications in the PSet or the Entity Documentation?

Reply:
I discussed with Juha H who didn't find the existing definitions suitable. However, if it's ok to generalize this and put any clarifications in the Pset documentation, I guess that could be ok (need to discuss with Juha first).

Reply:
@larswik yes i understand Juha's point, but it conflicts with the contraints laid out from bSI... 2 options arise:

-  change the name to NominalPavementThickness, then the documentation can stand as is.
- set the property documentation to: 
> The nominal thickness of the object. The size information is provided in addition to the shape representation and the geometric parameters used within. In case of inconsistency between the geometric parameters and the size properties, provided in the attached property set, the geometric parameters take precedence.

Then add the rest of the clarification in the relation between properties to the PSet.

Actually in this particular example think the second option is better as we shouldnt really be referencing other properties within the property description. the impact of one property on another is at the scope level of the Pset therefore should be in the PSet documentation.

@SergejMuhic thoughts....

PS there could be a third option of a Complex Property but i think that just over complicates the issue...